pkgs/development/libraries/glibc/default.nix

```diff
@@ -154,6 +154,8 @@ in
       mkdir -p $static/lib
       mv $out/lib/*.a $static/lib
       mv $static/lib/lib*_nonshared.a $out/lib
+      # If libutil.so.1 is missing, libutil.a is required.
+      test -f $out/lib/libutil.so.1 || mv $static/lib/libutil.a $out/lib
```

Comment:
Can you elaborate why it's needed in `$out` rather than `$static`?

Reply:
`libutil.so.1` is no longer provided and `libutil.a` is an empty static library for backward compatibility where `-lutil` is used in CFLAGS.

Reply:
See https://github.com/bminor/glibc/blob/5d00c201b9a2da768a79ea8d5311f257871c0b43/login/Makefile#L49-L62 for the relevant part in glibc.